Rewrite test files that have assertTrues instead of matchers

ryanhaining opened the pull request:
In the files I edited, I used `assertThat` and matchers for all assertions for the sake of uniformity. Some types of updates:

Container contents

```
// Before
Assertions.assertEquals(set.size, 3)
Assertions.assertTrue(set.contains("element1"))
Assertions.assertTrue(set.contains("second-elem"))
Assertions.assertTrue(set.contains("3rd"))
        
// After
assertThat(set).containsExactly( "element1", "second-elem", "3rd")
```

Container size

```
// Before
Assertions.assertEquals(true, c.reasonList.size == 2)

// After
assertThat(c.reasonList).hasSize(2) 
```

String contains

```
// Before
Assertions.assertTrue(c.getReasons(aapsLogger).contains("Running dev version. Closed loop is disabled."))

// After
assertThat(c.getReasons(aapsLogger)).contains("Running dev version. Closed loop is disabled.")
```

Float comparison

```
// Before
Assertions.assertEquals(2.0, c.value(), 0.01)

// After
assertThat(c.value()).isWithin(0.01).of(2.0)
```

Ordered comparison

```
// Before
Assertions.assertTrue(bolusForLowBg < bolusForBgInRange)

// After
assertThat(bolusForLowBg).isLessThan(bolusForBgInRange)
```

MilosKozak commented:
tests are failing
btw there are open PRs #2732 #2733 which are breaking tests. can you find the resson?

ryanhaining commented:
Sorry, somewhere between all tests passing and me committing, I added a stray `.` to `MidnightTimeTest`. I updated to remove it and reran locally with everything passing.
